assigner/commands/lock.py

```diff
@@ -15,6 +15,7 @@ def lock(args):
     #pylint: disable=no-value-for-parameter
     return manage_repos(
         args,
+        # TODO: Fix access
```

Comment:
Would the 'right' fix here be to have `RepoBase` (or maybe `StudentRepoBase`) implement `.lock()` and `.unlock()` methods and have the backend implementation handle the detail of what 'locking' and 'unlocking' means?

Reply:
I think so. Since GitHub would have a different access concept and API. I really think that  the backend should decide what access permission specific things need (or what unlocking / locking mean, if anything).